lib/tty/key.c

```diff
@@ -288,105 +296,41 @@ static key_define_t mc_default_keys[] = {
 
 /* Broken terminfo and termcap databases on xterminals */
 static key_define_t xterm_key_defines[] = {
+
+    /* The most common sequences are already covered by the Kitty Keyboard Protocol's backward
```

Comment:
i think the gist is "the kitty protocol extends the 'regular' keyboard protocol in a backwards compatible way. thus standard sequences are implicitly handled by the kitty handler, and don't need to be listed here".